src/tracks/MidiClip.cpp

```diff
@@ -174,19 +174,18 @@ TimePos MidiClip::beatClipLength() const
 
 	for (const auto& note : m_notes)
 	{
-		if (note->length() < 0)
+		if (note->type() == Note::StepNote)
 		{
 			max_length = qMax<tick_t>(max_length, note->pos() + 1);
 		}
 	}
 
-	if( m_steps != TimePos::stepsPerBar() )
+	if (m_steps != TimePos::stepsPerBar())
 	{
-		max_length = m_steps * TimePos::ticksPerBar() /
-						TimePos::stepsPerBar();
+		max_length = m_steps * TimePos::ticksPerBar() / TimePos::stepsPerBar();
 	}
 
-	return TimePos( max_length ).nextFullBar() * TimePos::ticksPerBar();
+	return TimePos(max_length).nextFullBar() * TimePos::ticksPerBar();
```

Comment:
Guidelines suggest to use ``{}`` rather than ``()`` to [avoid parsing ambiguities](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax) in the event that they may occur. Just a guideline that may not have any real impact in practice depending on the code, so make the switch if you so choose.